src/storage/resourceManager.test.ts

```diff
@@ -269,7 +284,7 @@ describe("ResourceManager kafka cluster methods", function () {
   });
 });
 
-describe("ResourceManager (CCloud) Schema Registry methods", function () {
```

Comment:
This block of tests was always testing topics methods, but was just grossly misnamed.